README.md

```diff
@@ -169,7 +169,7 @@ Running the game over internet currently requires setting up a relay.
   
 ### Relay Setup
  
-- Boss Room provides an integration with [Unity Relay](https://docs-multiplayer.unity3d.com/netcode/current/relay/). You can find our Unity Relay setup guide [here](https://docs.unity.com/ugs/en-us/manual/relay/manual/get-started)
+- Boss Room provides an integration with [Multiplayer Services sessions](https://docs.unity.com/ugs/en-us/manual/mps-sdk/manual) that make use of [Unity Relay](https://docs-multiplayer.unity3d.com/netcode/current/relay/). You can find our Unity Relay setup guide [here](https://docs.unity.com/ugs/en-us/manual/relay/manual/get-started)
```

Comment:
Same question as before: are there newer Sessions docs that we can point to or does it still make sense to point to Relay docs?

Reply:
I don't think it's relevant either. We can just rephrase this to mean we utilized the Multiplayer SDK and its underlying Sessions API.